doc/nrf/doc_styleguide.rst

```diff
@@ -185,31 +236,53 @@ Functions
 
 #. **@return** should be used to describe a return value (for example, "@return The length of ...", "@return The handle").
    There is usually only one return value.
+
+   .. code-block:: c
+      :caption: Return documentation example
+
+        *  @return  Initializer that sets up the pipe, length, and byte array for
+        *           content of the TX data.
+
 #. **@retval** should be used for specific return values (for example, "@retval true", "@retval CONN_ERROR").
    Describe the condition for each of the return values (for example, "If the function completes successfully", "If the connection cannot be established").
    If there is only one retval, add what happens otherwise. Example: "Otherwise, an error code is returned".
+
+    .. code-block:: c
+       :caption: Retval documentation example
+
+         *  @retval 0 If the operation was successful.
+         *            Otherwise, a (negative) error code is returned.
+         *  @retval (-ENOTSUP) Special error code used when the UUID
+         *            of the service does not match the expected UUID.
+
 #. **Do not use @returns**.
```

Comment:
Again, I think it's totally enough to put only the doxygen element in bold, not whole part of sentence.